NEWS.md

```diff
@@ -165,6 +165,13 @@ Outstanding ones only.
         * Symbol#to_proc now returns a lambda Proc.
           [[Feature #16260]]
 
+* Warning
+
+    * Modified method
+
+        * Warning#warn now supports a category kwarg.
```

Comment:
@eileencodes `warn` is a singleton method. How about changing the name to `Warning.warn` ?

Reply:
Technically, it's not a singleton method, it's a method in a module that extends the module.  `Warning#warn` is an instance method, and `Warning` extends itself.  So it is correct as written.

Reply:
@jeremyevans Oh, very complicated, but I understand. Thank you for your explanation!